docs/_posts/examples/3400-01-08-html-popup.html

```diff
@@ -0,0 +1,104 @@
+---
+layout: example
+category: example
+title: Custom HTML popups
+description: Create popups when markers are clicked
+permalink: /examples/html-popups
+---
+
+<div id='map'></div>
+<script>
+mapboxgl.util.getJSON('https://www.mapbox.com/mapbox-gl-styles/styles/basic-v7.json', function (err, style) {
+  if (err) throw err;
+
+  //add a new stylesheet layer to anticipate our markers
+  style.layers.push({
+    "id": "markers",
+    "type": "symbol",
+    "source": "markers",
+    "interactive":true,
+    "layout": {
+      "icon-image": "dot.sdf"
+    },
+    "paint": {
+      "icon-color": "#993379",
+      "icon-halo-color":"white",
+      "icon-halo-width":2
+    }
+  });
+
+  // initiate the map
+  var map = new mapboxgl.Map({
+    container: 'map',
+    style: style,
+    center: [27.8, -96],
+    zoom: 1
+  });
+
+  // data for our markers
+  var geoJSON = {
+    "type": "FeatureCollection",
+    "features": [
+      {
+        "type": "Feature",
+        "geometry": {
+          "type": "Point",
+          "coordinates": [-77.03238901390978, 38.913188059745586]
+        },
+        "properties": {
+          "title": "Mapbox DC",
+          "address1": "1714 14th St NW, Rear Entrance",
+          "address2": "Washington, DC 20009"
+        }
+      },
+      {
+        "type": "Feature",
+        "geometry": {
+          "type": "Point",
+          "coordinates": [-122.414, 37.776]
+        },
+        "properties": {
+          "title": "Mapbox SF",
+          "address1": "149 9th St, Suite 404",
+          "address2": "San Francisco, CA 94103"
+        }
+      },
+      {
+        "type": "Feature",
+        "geometry": {
+          "type": "Point",
+          "coordinates": [-74.224812, -13.160310]
+        },
+        "properties": {
+          "title": "Mapbox Ayacucho",
+          "address1": "Jr. José Santos Chocano Nº 416",
+          "address2": "Jesús Nazareno, Ayacucho"
+        }
+      }
+    ]};
+
+  // add each point's coordinates as a property value, to be used for popup positioning
+  geoJSON.features
+    .forEach(function(n){
+      n.properties.coordinates={"lat": n.geometry.coordinates[1],"lng":n.geometry.coordinates[0]};
+  	});
+  // define and bind the data to our "markers" layer in the stylesheet
+  var markers = new mapboxgl.GeoJSONSource({ data: geoJSON });
+  map.addSource('markers', markers);
+
+  //set a click listener on the map object, to produce a popup whenever a click registers near a marker
+  map.on('click', function(e) {
+      map.featuresAt(e.point, {radius: 15}, function(err, features) {
+          if (err) throw err;
+          if (features[0]) {
+          	var markerPosition = features[0].properties.coordinates;
+          	var popupText = '<strong>'+features[0].properties.title+'</strong><br>'+ features[0].properties.address1+'<br>'+features[0].properties.address2;
```

Comment:
We should set a good example here and correctly HTML-escape the interpolated properties.